app/helpers/organizations_helper.rb

```diff
@@ -0,0 +1,14 @@
+module OrganizationsHelper
+  def can_edit_user_role?(current_user, member, current_organization)
+    user_role = current_user.role_for(current_organization)
+    member_role = member.role_for(current_organization)
+
+    ((user_role == "owner" && member_role.in?(["developer", "viewer"])) ||
+     (user_role == "developer" && member_role == "viewer")) &&
+      current_user.id != member.id
+  end
```

Comment:
**suggestion**: I don't think this will be about any other user or any other organization in the near future, we can simplify this and just assume `current_user` and `current_organization` (the helpers) and then rely on `current_user_role` directly – `can_current_user_edit_role?(member)`